<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetturnover</class>
 <widget class="QWidget" name="widgetturnover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QComboBox" name="comboBoxRange">
     <item>
      <property name="text">
       <string>日流水</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>周流水</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>月流水</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>半年流水</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>年流水</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableViewOrder"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QtMaterialRaisedButton" name="pushButtonFrontPage">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>首页</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_17">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QtMaterialRaisedButton" name="pushButtonPrevPage">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>上一页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>当前页：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>第</string>
             </property>
             <property name="margin">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxPageJump">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <bold>true</bold>
              </font>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>页</string>
             </property>
             <property name="margin">
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>/</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>共</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LabelTotalPages">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="font">
              <font>
               <family>微软雅黑</family>
              </font>
             </property>
             <property name="text">
              <string>页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QtMaterialRaisedButton" name="pushButtonNextPage">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>下一页</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_18">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QtMaterialRaisedButton" name="pushButtonBackPage">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>尾页</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>订单量：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="amount">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>总流水：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
